rescript/trim_alignment.py

```diff
@@ -220,7 +220,9 @@ def _trim_alignment(expand_alignment_action,
 
         # expand the existing alignment by addition of primers
         alignment_with_primers = expand_alignment_action(
```

Comment:
the more q2-canonical way to write this would be:
```
         alignment_with_primers, = expand_alignment_action(
            alignment=aligned_sequences,
            sequences=primers,
            addfragments=True)
```